react/features/base/logger/reducer.js

```diff
@@ -0,0 +1,42 @@
+/* @flow */
+
+import { ReducerRegistry } from '../redux';
+
+import {
+    SET_LOG_COLLECTOR,
+    START_LOG_COLLECTOR
+} from './actionTypes';
+
+const INITIAL_STATE = {
+    /**
+     * The log collector which captures JS console logs for this app.
+     * @type {LogCollector}
+     */
+    logCollector: null,
+
+    /**
+     * Indicates if the log collector has been started (it will not be started
+     * if the welcome page is displayed).
+     */
+    logCollectorStarted: false
```

Comment:
If the `INITIAL_STATE` is composed only of properties with values practically equal to undefined, then what's the purpose of defining them at all? Please remove them.